stan/math/rev/fun/cholesky_decompose.hpp

```diff
@@ -87,7 +89,7 @@ inline auto cholesky_lambda(T1& L_A, T2& L, T3& A) {
     using Eigen::Lower;
     using Eigen::StrictlyUpper;
     using Eigen::Upper;
-    Eigen::MatrixXd L_adj(L.rows(), L.cols());
+    Eigen::MatrixXd L_adj = Eigen::MatrixXd::Zero(L.rows(), L.cols());
```

Comment:
Oh interesting, was the issue here that the upper triangular part of the matrix was not being set?

Reply:
Actually I meant to leave a comment there. That one change I just did cause it looked scary and was too lazy to check the Eigen docs or carefully read the code.

Reply:
That one can def. be reverted but I just saw it and got nervous.